src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/config/AuditConfig.java

```diff
@@ -286,10 +245,16 @@ public String toString() {
                     ", resolveIndices=" + resolveIndices +
                     ", excludeSensitiveHeaders=" + excludeSensitiveHeaders +
                     ", ignoredAuditUsers=" + ignoredAuditUsersMatcher +
-                    ", ignoredComplianceUsersForRead=" + ignoredComplianceUsersForReadMatcher +
-                    ", ignoredComplianceUsersForWrite=" + ignoredComplianceUsersForWriteMatcher +
                     ", ignoreAuditRequests=" + ignoredAuditRequestsMatcher +
                     '}';
         }
     }
+
+    public static Set<String> getSettingAsSet(final Settings settings, final String key, final List<String> defaultList, final boolean ignoreCaseForNone) {
```

Comment:
Is it possible to move this to a common util class, since this seems to be used by both AuditConfig and ComplianceConfig?

Reply:
Should `ComplianceConfig` be moved to `AuditConfig`? Looks like `Filter` and `Compliance` are both related to `Audit`.

Reply:
There will be a `Compliance` to represent storage in index and generate ComplianceConfig out of it. Dependency on settings is stopping from directly using it unless there is some workaround. 
This is where I want to get to https://github.com/opendistro-for-elasticsearch/security/blob/5a81a60cc05d37d92a965e8cb03a5214c10e03c1/src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/config/AuditConfig.java 

Hence thought of doing it separate PRs.

Reply:
Consider cutting a separate (preliminary) PR to move `ComplienceConfig` to `AuditConfig` if that will help to address @debjanibnrj comment.

Reply:
Referencing @debjanibnrj comment I don't think there needs to be a separate util class as it can be just a util function within AuditConfig and the function itself might disappear with  #325